### ISSUE TITLE
Add GitLab identity for signing in

### ISSUE THREAD

**Issue by hason** [gitterHQ/gitter#1429]
Add support for signing in with GitLab.

Related issue https://github.com/gitterHQ/gitter/issues/273

**Comment by MadLittleMods** [comment]
To clarify your description:

 - Are you looking for a way to sign in with GitLab?
 - The link you are referring to would be in the chat heading and repo-info in the right sidebar like we have now for GitHub repos?

**Comment by hason** [comment]
@MadLittleMods Yes, I am looking for a way to sign in to Gitter with GitLab.com account.

**Comment by MadLittleMods** [comment]
> Can I continue to use Gitter with GitHub accounts and projects?
> 
> Absolutely. Login with GitHub and Twitter will continue to work, as well as all of the integrations we have today with GitHub issues and other services. We have had a GitLab integration for a long time and will improve this by adding the ability to login with your GitLab.com account.
> 
> http://blog.gitter.im/2017/03/15/gitter-gitlab-acquisition/
